client/packages/invoices/src/InboundShipment/DetailView/Toolbar.tsx

```diff
@@ -115,6 +128,14 @@ export const Toolbar: FC = () => {
             <DropdownMenuItem IconComponent={DeleteIcon} onClick={onDelete}>
               {t('button.delete-lines')}
             </DropdownMenuItem>
+            {isInvoiceCreatedBeforeStore && (
+              <DropdownMenuItem
+                IconComponent={RewindIcon}
```

Comment:
Wasn't too sure about which icon to use... couldn't find the broom icon on feature and Mark suggested rewind icon so went with that 🤷🏻

Reply:
![image](https://github.com/msupply-foundation/open-msupply/assets/7684221/9f5e80d5-226f-4cab-bd94-e261dba347d1)

Can't find any feather one but there is broom sort of... I think I like rewind more though.